frontend/app/controllers/user-home.js

```diff
@@ -170,7 +171,17 @@ export default Controller.extend({
             } else if (actionDefinition.success.type === 'display') {
               $(`#${actionDefinition.success.target}`).text(resultValue);
             } else if (actionDefinition.success.type === 'update-plot') {
-              $.jqplot(actionDefinition.success.target, resultValue).replot();
+              if (actionDefinition.success.stacking === 'xy') {
+                // alert(JSON.stringify($(`#${actionDefinition.success.target}`).data(), null, 4));
+                // $(`#${actionDefinition.success.target}`).data('jqplot').replot({ data: [_.zip.apply(null, resultValue)] });
+                $.jqplot(actionDefinition.success.target, [_.zip.apply(null, resultValue)]).replot();
```

Comment:
Why do you need to use `Function.prototype.apply` when you could directly call the function ? What's the reason ?